sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/ScoringParameter.java

```diff
@@ -21,7 +21,7 @@
  * Represents a parameter value to be used in scoring functions (for example, referencePointParameter).
  */
 public final class ScoringParameter {
-    private final ClientLogger logger = new ClientLogger(ScoringParameter.class);
+    private static final ClientLogger LOGGER = new ClientLogger(ScoringParameter.class);
```

Comment:
Is this field not going on the wire? It does not have JsonIgnore on it?

The other two model types SearchResults, SuggestResult seem to be responses so I think those don't need it.

Reply:
This goes over the wire but is handled manually as it needs to have a specific conversion applied